STEERING-COMMITTEE.md

```diff
@@ -139,24 +130,61 @@ Quorum is considered reached when at least half of the members are present.
 
 ## Getting in touch
 
-There are two ways to raise issues to the steering committee for decision:
+There are three ways to raise issues to the steering committee for decision:
 
 1. Emailing the steering committee at
    [steering@knative.team](mailto:steering@knative.team).
    This is a private discussion list to which all members of the committee have
    access.
 2. Open an issue on a knative repository and indicate that you would like
    attention from the steering committee.
+   
+3. Asking a question on the
+   [#knative-toc-sc-questions](https://cloud-native.slack.com/archives/C04LQCW0C03)
+   channel on [CNCF Slack](https://cloud-native.slack.com/).
 
 ## Composition
 
-The Steering Committee has five (5) seats. Of these five seats, four (4) are elected
-Contributing seats, open to any project contributor, and one (1) is an
-[End User Seat](#end-user-seat) appointed by the Steering Committee.
+The Steering Committee has between five (5) and seven (7) members. These seats are
+composed as follows:
+
+- Four (4) are elected Contributing Seats, open to any project contributor.
+  Two (2) Contributing Seats are elected every year, and serve a two-year term.
+  In the event of a vacancy, the Steering Committee may select a replacement to
+  serve out the remainder of the term.
+
+- One (1) is an [End User Seat](#end-user-seat) appointed by the Steering
+  Committee.  The End User Seat serves until the next elected Steering Committee
+  appoints a replacement in the two months after members are seated, as
+  described below.
+
+- Up to two Selected Seats (2) are appointed by the Steering Committee to
+  balance the set of skills on the Steering Committee.  These seats serve a
+  1-year term, until Contributing Seats for the next Steering Committee election
+  are seated.
 
 Steering Committee contributing members serve for 2-year terms, staggered in
-order to preserve continuity. End user member serves for 1-year term. Every year
-two contributor seats are elected.
+order to preserve continuity. End user and selected members serve for 1-year
+terms. Every year two contributor seats are elected.
+
+### Limitations on Company Representation
+
+No more than two contributing seats may be held by employees of the same organization
+(or conglomerate, in the case of companies owning each other). Additionally, the end
+user seat must belong to an employee from a different organization. No company may
+hold a majority (fifty percent or more) of the total Steering Committee seats through
+any combination of contributing, end-user, and selected seats. If the results of an
```

Comment:
```suggestion
any combination of seats. If the results of an
```

Reply:
removed the incomplete list of potential seats to clarify it relates to all types of seats.